ethexe/consensus/src/validator/producer.rs

```diff
@@ -324,7 +323,7 @@ mod tests {
     #[tokio::test]
     async fn code_commitments_only() {
         let (ctx, keys) = mock_validator_context();
-        let validators = vec![ctx.pub_key.to_address(), keys[0].to_address()];
+        let validators = nonempty::nonempty![ctx.pub_key.to_address(), keys[0].to_address()];
```

Comment:
```suggestion
        let validators = nonempty![ctx.pub_key.to_address(), keys[0].to_address()];
```
and in all other places